lib/game/Minesweeper/components/info.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter/material.dart';
+import '../management/gamelogic.dart';
+import '../management/gametimer.dart';
+import '../theme/colors.dart';
+
+class GameInfo extends ConsumerWidget{
+  const GameInfo({super.key, required this.resetGame, required this.timer});
+  final void Function() resetGame;
+  final GameTimer timer;
+  final double opacityValue = 0.8;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    // Time Over
+    if(timer.getTime() == "00:00"){
```

Comment:
It's more meaningful to let `getTime()` return a `Duration` object instead of a string if you want to check if the timer gets zero.